packages/frontend-2/components/projects/Dashboard.vue

```diff
@@ -113,7 +113,7 @@ const {
   filter: {
     search: (search.value || '').trim() || null,
     onlyWithRoles: selectedRoles.value?.length ? selectedRoles.value : null,
-    workspaceId: isWorkspaceNewPlansEnabled ? (null as Nullable<string>) : undefined
+    personalOnly: isWorkspaceNewPlansEnabled.value
```

Comment:
there's no such thing as undefined in GQL, its either a set value or null. so the current implementation of "only get personal projects" is busted, which is why I introduced a properly working flag